src/CSharp/MetadataWebApi/MetadataWebApi/MetadataWebApi.csproj

```diff
@@ -1,118 +1,23 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Yeah, I'll update the docs to say that either Visual Studio 2017, Visual Studio Code or dotnet command line is now required.